src/app.js

```diff
@@ -38,7 +38,8 @@ import defaultMessages from './translations/en.json'
 
 // Step 3:
 // If you are using a non-english locale, point `messagesInLocale` to correct .json file
-import messagesInLocale from './translations/en.json' // TODO
+import messagesInLocale from './translations/en.json'
+import { ServiceTypeProvider } from './context/ServiceTypeProvider' // TODO
```

Comment:
Did you do the TODO? Should we make a ticket for it?

Reply:
No need for TODO tickets, just a reminder for me, removing them